Python/README.md

```diff
@@ -1,4 +1,8 @@
-#Python
+##Python Samples
+
+[![Build Status](https://travis-ci.org/alfpark/azure-batch-samples.svg?branch=master)](https://travis-ci.org/alfpark/azure-batch-samples)
```

Comment:
For windows based builds, we should use AppVeyor: http://www.appveyor.com/

Travis and AppVeyor can be used SxS.